src/pages/ActivityList/Activity.js

```diff
@@ -0,0 +1,42 @@
+import {AiFillStar} from 'react-icons/ai';
+import {useState} from 'react';
+
+export default function Activity(props) {
+  const [isFavorite, setIsFavorite] = useState(false);
+
+  let formatPhoneNumber = (str) => {
+    //Filter only numbers from the input
+    let cleaned = ('' + str).replace(/\D/g, '');
+
+    //Check if the input is of correct
+    let match = cleaned.match(/^(1|)?(\d{3})(\d{3})(\d{4})$/);
+
+    if (match) {
+      //Remove the matched extension code
+      //Change this to format for any country code.
+      let intlCode = (match[1] ? '+1 ' : '')
+      return [intlCode, '(', match[2], ') ', match[3], '-', match[4]].join('')
+    }
+
+    return null;
+  }
+
+  var activityObj = props.action;
+  return (
+    <div className="activity-widget">
+      {isFavorite ? <AiFillStar color="gold" size="36" onClick={() => setIsFavorite(!isFavorite)}/> : <AiFillStar color="white" size="36" onClick={() => setIsFavorite(!isFavorite)}/>}
```

Comment:
Consider using variables from a file for inline css so you can reuse it or at least not have to hunt around and change it in different places